tmt/steps/report/html.py

```diff
@@ -76,8 +79,8 @@ def go(self) -> None:
 
         environment.trim_blocks = True
         environment.lstrip_blocks = True
-        with open(HTML_TEMPLATE_PATH) as file:
-            template = environment.from_string(file.read())
+        with importlib.resources.as_file(HTML_TEMPLATE_PATH) as path:
+            template = environment.from_string(path.read_text())
```

Comment:
I'm not sure I understand the benefit here. Should this be just this?

```python
template = environment.from_string(HTML_TEMPLATE_PATH.read_text())
```

As `HTML_TEMPLATE_PATH` is already a `Path` instance, right?

Reply:
nice catch, thanks.